iina/Base.lproj/MainMenu.xib

```diff
@@ -264,12 +264,12 @@
                             <menuItem title="Volume:" enabled="NO" id="iQF-1S-9EK">
                                 <modifierMask key="keyEquivalentModifierMask"/>
                             </menuItem>
-                            <menuItem title="Volume + 5%" tag="10" keyEquivalent="" id="Wbq-vt-hzX"/>
-                            <menuItem title="Volume + 1%" tag="1" alternate="YES" keyEquivalent="" id="c6Y-0e-vgf">
+                            <menuItem title="Increase Volume 5%" tag="10" keyEquivalent="" id="Wbq-vt-hzX"/>
+                            <menuItem title="Increase Volume 1%" tag="1" alternate="YES" keyEquivalent="" id="c6Y-0e-vgf">
```

Comment:
I'd expect that this would mean that the translators will have to update translations? I'm not quite sure how you'd like me to do this.